smartmin/users/context_processors.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import unicode_literals
+from django.conf import settings
+
+
+def link_components(request):
```

Comment:
This should just be in /email.py ..

Having it in context_processors is going to be super confusing as people would expect it to work on actual requests, not email requests.

Don't forget to change the signature to also take user.